longturn/freeciv21
# Make sprite getters return const QPixmap*

Functions that retrieve sprites from the tileset used to return a non-const
pointer, which allowed other code to change them. This resulted in #745.
This commit changes all sprite getters to return const QPixmap* instead of
QPixmap*, solving #745 and preventing it from happening in the future.

Maybe this should've been done in smaller commits but it's too late for that.

Closes #745.

Test plan (from #745 instructions):

*    Use city style "simple"
*    Found a city
*    Do F3
*    Make sure the map at your city gets redrawn (for instance by scrolling away)
*    See error
